cardano-cli/test/cardano-cli-golden/files/input/governance/costmodels.json

```diff
@@ -0,0 +1,573 @@
+    "lessThanEqualsInteger-cpu-arguments-slope": 1366,
+    "fstPair-memory-arguments": 32,
+    "modInteger-memory-arguments-intercept": 0,
+    "unConstrData-cpu-arguments": 150000,
+    "lessThanEqualsInteger-memory-arguments": 1,
+    "chooseUnit-memory-arguments": 32,
+    "sndPair-memory-arguments": 32,
+    "addInteger-cpu-arguments-intercept": 197209,
+    "decodeUtf8-memory-arguments-slope": 8,
+    "equalsData-cpu-arguments-intercept": 150000,
+    "mapData-cpu-arguments": 150000,
+    "mkPairData-cpu-arguments": 150000,
+    "quotientInteger-cpu-arguments-constant": 148000,
+    "consByteString-memory-arguments-slope": 1,
+    "cekVarCost-exBudgetMemory": 100,
+    "indexByteString-cpu-arguments": 150000,
+    "unListData-cpu-arguments": 150000,
+    "equalsInteger-cpu-arguments-slope": 1326,
+    "cekStartupCost-exBudgetMemory": 100,
+    "subtractInteger-cpu-arguments-intercept": 197209,
+    "divideInteger-cpu-arguments-model-arguments-intercept": 425507,
+    "divideInteger-memory-arguments-intercept": 0,
+    "cekForceCost-exBudgetMemory": 100,
+    "blake2b-cpu-arguments-intercept": 2477736,
+    "remainderInteger-cpu-arguments-constant": 148000,
+    "tailList-cpu-arguments": 150000,
+    "encodeUtf8-cpu-arguments-intercept": 150000,
+    "equalsString-cpu-arguments-slope": 1000,
+    "lessThanByteString-memory-arguments": 1,
+    "multiplyInteger-cpu-arguments-slope": 11218,
+    "appendByteString-cpu-arguments-intercept": 396231,
+    "lessThanEqualsByteString-cpu-arguments-slope": 248,
+    "modInteger-memory-arguments-slope": 1,
+    "addInteger-cpu-arguments-slope": 0,
+    "equalsData-cpu-arguments-slope": 10000,
+    "decodeUtf8-memory-arguments-intercept": 0,
+    "chooseList-cpu-arguments": 150000,
+    "constrData-cpu-arguments": 150000,
+    "equalsByteString-memory-arguments": 1,
+    "cekApplyCost-exBudgetCPU": 29773,
+    "quotientInteger-memory-arguments-slope": 1,
+    "verifySignature-cpu-arguments-intercept": 3345831,
+    "unMapData-memory-arguments": 32,
+    "mkCons-memory-arguments": 32,
+    "sliceByteString-memory-arguments-slope": 1,
+    "sha3_256-memory-arguments": 4,
+    "ifThenElse-memory-arguments": 1,
+    "mkNilPairData-memory-arguments": 32,
+    "equalsByteString-cpu-arguments-slope": 247,
+    "appendString-cpu-arguments-intercept": 150000,
+    "quotientInteger-cpu-arguments-model-arguments-slope": 118,
+    "cekApplyCost-exBudgetMemory": 100,
+    "equalsString-memory-arguments": 1,
+    "multiplyInteger-memory-arguments-slope": 1,
+    "cekBuiltinCost-exBudgetMemory": 100,
+    "remainderInteger-memory-arguments-intercept": 0,
+    "sha2_256-cpu-arguments-intercept": 2477736,
+    "remainderInteger-cpu-arguments-model-arguments-intercept": 425507,
+    "lessThanEqualsByteString-memory-arguments": 1,
+    "tailList-memory-arguments": 32,
+    "mkNilData-cpu-arguments": 150000,
+    "chooseData-cpu-arguments": 150000,
+    "unBData-memory-arguments": 32,
+    "blake2b-memory-arguments": 4,
```

Comment:
why is this without the bitsize, i.e. not `blake2b_256-`?

Reply:
https://github.com/input-output-hk/cardano-ledger/pull/3367/files#diff-f481d9eb9f793235d9fc4a1965ec1e0a01bed313f90adb11dcd9aa099a4cd738R371